src/css/hero.css

```diff
@@ -1,159 +1,504 @@
+source {
+  display: block;
+}
+body {
+  box-sizing: border-box;
+  font-family: 'DM Sans', sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+  color: #fff;
+  background-color: #ae1332;
+}
+.h-container {
+  margin: 0px auto;
+}
+
+@media screen and (max-width: 374px) {
+  .h-container {
+    width: 100%;
+  }
+}
+
+@media screen and (min-width: 375px) {
+  .h-container {
+    max-width: 375px;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .h-container {
+    max-width: 768px;
+  }
+}
+
 @media screen and (min-width: 1280px) {
+  .h-container {
+    max-width: 1280px;
+  }
+}
+
+.hero {
+  position: relative;
+}
+
+.hero-container {
+  background-image: url(../img/mobilestrawberries.png),
+    url(../img/mobilebananas.png);
+  background-repeat: no-repeat;
+  background-size: 56px 56px, 66px 44px;
+  background-position: top 56px right 0px, top 116px left -7px;
+}
+
+@media (-webkit-min-device-pixel-ratio: 2),
+  (min-resolution: 192dpi),
+  (min-resolution: 2dppx) {
+  .hero-container {
+    background-image: url(../img/mobilestrawberries@2x.png),
+      url(../img/mobilebananas@2x.png);
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .hero-container {
+    background-image: url(../img/tabletstrawberries.png),
+      url(../img/tabletbananas.png);
+    background-size: 108px 108px, 132px 90px;
+    background-position: top 54px right -5px, top 169px left -5px;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  @media (min-device-pixel-ratio: 2),
+    (min-resolution: 192dpi),
+    (min-resolution: 2dppx) {
+    .hero-container {
+      background-image: url(../img/tabletstrawberries@2x.png),
+        url(../img/tabletbananas@2x.png);
+    }
+  }
+}
+
+@media only screen and (min-width: 1280px) {
+  .hero-container {
+    background-image: url(../img/desktopstrawberries.png),
+      url(../img/desktopbananas.png);
+    background-size: 120px 120px, 183px 125px;
+    background-position: top 39px right 191px, top 191px left 181px;
+  }
+}
+
+@media only screen and (min-width: 1280px) {
+  @media (min-device-pixel-ratio: 2),
+    (min-resolution: 192dpi),
+    (min-resolution: 2dppx) {
+    .hero-container {
+      background-image: url(../img/desktopstrawberries@2x.png),
+        url(../img/desktopbananas@2x.png);
+    }
+  }
+}
+.header-primary {
+  padding-top: 56px;
```

Comment:
padding nie potrzebny.

Reply:
to ten padding, który chcesz mieć w hero ale tam nie mogę go dodać bo pozycjonuję różne elementy względem siebie i wszystko się rozjedzie